deadlock_examples/Account.rb

```diff
@@ -0,0 +1,29 @@
+class Account 
+  include Celluloid
+
+  def initialize
+    @mutex = Mutex.new
+    @balance = 500
+  end
+
+  def withdraw(amount)
+   @balance = @balance - amount;
+ end
+
+ def deposit(amount)
+   @balance = @balance + amount;
+ end
+
+ def transfer(from, to, amount)
+  @mutex.lock
+  from.withdraw(amount)
+  to.deposit(amount)
+  @mutex.unlock
+  end
+end
+
+account1 = Account.new
```

Comment:
Your specs go in `spec/celluloid/group/unlocker/*` and your examples go in `spec/shared/unlocker/*` ... the files are good, just change their location and avoid instantiating classes outside your tests.